ehitchhiking-ui/src/app/pages/registration/registration.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "page">
+<div id="login-form">
+    <div class = "exadel-login">
+      <img src="../../../assets/images/FullColor_Word-1.png" alt="exadel-logo">
+    </div>
+
+  <fieldset class = "f-set">
+    <form class = "form-img">
+      <form (submit)="onSubmit($event)">
+        <div class = "icon-inp">
+          <i class="material-icons">
+            perm_identity
+          </i>
+          <input type="login" class = "input" [(ngModel)]="login" name = "login" required placeholder="Login" >
```

Comment:
Don't forget to set / in unclosed tags
example: <input type="login" class = "input" [(ngModel)]="login" name = "login" required placeholder="Login" />

Reply:
okay